pkg/traceql/enum_attributes.go

```diff
@@ -89,6 +89,13 @@ const (
 
 	IntrinsicTraceID
 	IntrinsicSpanID
+	IntrinsicSpanStatus
```

Comment:
To differentiate perhaps we can do "ScopedIntrinsic-" instead of continuing the "Intrinsic" pattern? What do you think?

Reply:
That sounds good to me, I'll make the change!

Reply:
Made the changes, Requesting re-review.